scipy/stats/_stats.pyx

```diff
@@ -513,6 +513,113 @@ cdef double _geninvgauss_pdf(double x, void *user_data) nogil except *:
     return math.exp(_geninvgauss_logpdf_kernel(x, p, b))
 
 
+cdef double _phi(double z) nogil:
+    """evaluates the normal PDF. Used in `studentized range`"""
+    cdef double inv_sqrt_2pi = 0.3989422804014327
+    return inv_sqrt_2pi * math.exp(-0.5 * z * z)
+
+
+cdef double _logphi(double z) nogil:
+    """evaluates the log of the normal PDF. Used in `studentized range`"""
+    cdef double log_inv_sqrt_2pi = -0.9189385332046727
+    return log_inv_sqrt_2pi - 0.5 * z * z
+
+
+cdef double _Phi(double z) nogil:
+    """evaluates the normal CDF. Used in `studentized range`"""
+    # use a custom evaluation because using cs.ndtr results in incorrect PDF
+    # evaluation at q=0 on 32bit systems
+    cdef double m_sqrt1_2 = 0.7071067811865475
```

Comment:
Let's rename this to `inv_sqrt_2` for consistency with about and mention why we can't use `M_SQRT1_2`